packages/saltcorn-admin-models/models/backup.ts

```diff
@@ -374,8 +374,9 @@ const backup_info_file = async (root_dirpath: string): Promise<void> => {
 const zipFolder = async (folder: string, zipFileName: string) => {
   const backup_with_system_zip = getState().getConfig(
     "backup_with_system_zip",
-    false
```

Comment:
Default should still be false here. ALTERNATIVELY: remove this config setting entirely, replace with executableIsAvailable("zip") - this is better